### DISCUSSION TITLE
Bitdefender doesn't like astroidframe.work

### DISCUSSION THREAD

**Discussion by fontanil** [templaza/astroid-framework#411] [General]
Hi,

Bitdefender has been blocking astroid.framework for a long time now

I French "La page https://astroidframe.work/ a été identifiée comme ayant des activités suspectes. Il n'est pas recommandé de poursuivre la navigation sur ce site Web."

Translated with DeepL : "The https://astroidframe.work/ page has been identified as having suspicious activity. Further browsing of this website is not recommended."

Regards,
Robert

**Comment by Chacapamac** [comment]
No problems with U-Block and Ghostery, the only flag seem to be on googletagmanager.com

**Comment by Chacapamac** [comment]
Can I close this discussion?

**Comment by fontanil** [comment]
If you want, but Bitdefender says in French:

> La page https://astroidframe.work/ a été identifiée comme ayant des activités suspectes. Il n'est pas recommandé de poursuivre la navigation sur ce site Web

**Comment by Chacapamac** [comment]
OK, this is not good, **Bitdefender** is widely use on Mac... I keep it as an issue....

**Comment by Banuk1616** [comment]
I also use Bitdefender and get the same message with fontanil every time I visit the Astroid Framework website.
Best regards Dirk

**Comment by sonvnn** [comment]
I wonder if deleting Google Tag Manager will resolve this issue? Can you help me come up with a solution for this problem?

Thanks & Best Regards,
Sonny

**Comment by fontanil** [comment]
Maybe you should ask Bitdefender about the reason for the alert?

**Comment by SoundHunter1** [comment]
You can report a false positive to Bitdefender here: https://www.bitdefender.com/consumer/support/answer/29358/
Mostly they react quickly on them.
Best if the website owner send the report in.

**Reply by sonvnn** [reply]
Thank you! I sent them a ticket.

**Comment by Banuk1616** [comment]
I still get the message from Bitdefender every time I visit the Astroid Framework page. Has Bitdefender already replied to the ticket? It is not a good feeling when the security program warns you every time you visit the Astroid Framework site.

**Reply by sonvnn** [reply]
I received no response from them. Could you please help me with your feedback on this issue? More people making requests will probably make things happen faster.

**Comment by fontanil** [comment]
Hi, I opened a ticket this morning and Bitdefender got back to me very quickly with that link https://www.bitdefender.co.uk/consumer/support/answer/88485/
In French https://www.bitdefender.fr/consumer/support/answer/10905/

**Comment by Banuk1616** [comment]
Hi Sonny, I opened two tickets with Bitdefender this morning, one on my wife's email address and one on my email address. When I visited the Astroid website just now, I no longer received a warning.

@fontanil
can you confirm that the warning is gone?

**Reply by sonvnn** [reply]
Thank you all. It makes sence :)

**Comment by fontanil** [comment]
Yes, I have no warning now :-)

**Reply by sonvnn** [reply]
Thank you very much. It makes sence :)
